data/map.txt

```diff
@@ -29434,6 +33653,11 @@ planet Furnace
 		threshold 3500
 		fleet "Large Syndicate" 7
 
+planet "Gara Pvu"
+	landscape land/sky3
+	description `Beautiful landscapes cover this planet, and unlike nearby systems, the climate is more than welcoming to a human being. Most remarkably, there are small selections of elegant curved structures and overpasses spread out in patches along the surface. While they are fantastical in appearance, they are unmistakably not part of the land itself, and must have been left long ago by some other species that also managed to leeave the natural beauty of this planet intact, or it's been so long that nature has retaken everything.`
```

Comment:
```suggestion
	description `Beautiful landscapes cover this planet, and unlike nearby systems, the climate is more than welcoming to a human being. Most remarkably, there are small selections of elegant curved structures and overpasses spread out in patches along the surface. While they are fantastical in appearance, they are unmistakably not part of the land itself, and must have been left long ago by some other species that also managed to leave the natural beauty of this planet intact - either that, or it's been so long since their disappearance that nature has retaken everything.`
```